templates/admin/manage_orders.html

```diff
@@ -0,0 +1,107 @@
+{% extends "base.html" %}
+{% block head %}<script src="../../static/js/order.js"></script>{% endblock %}
+{% block content %}
+    {% set ns = namespace() %}
+    {% set ns.total_cost=0 %}
+    {% set ns.total_nr_items=0 %}
+    {% set ns.items = orders %}
+    <div class="columns is-centered">
+        {% if ns.items|length > 0 %}
+            <div class="column is-6 has-text-centered">
+                {% for order_nr in ns.items %}
+                    {% set ns.background = "has-background-grey-light" %}
+                    {% set ns.status = "is being packaged" %}
+                    {% if ns.items[order_nr][0].status == 1 %}
+                        {% set ns.background = "has-background-info" %}
+                        {% set ns.status = "is in route to you" %}
+                    {% elif ns.items[order_nr][0].status == 2 %}
+                        {% set ns.background = "has-background-success" %}
+                        {% set ns.status = "has been delivered" %}
+                    {% elif ns.items[order_nr][0].status == 3 %}
+                        {% set ns.background = "has-background-danger" %}
+                        {% set ns.status = "has been refunded" %}
+                    {% endif %}
+                    <div class="box is-center has-text-centered {{ ns.background }} ">
+                        <nav class="level is-mobile">
+                            <div class="level-left">
+                                Order number&nbsp; <b>{{ order_nr }}</b>
+                            </div>
+                            <div class=""level-right>
+                                {% if ns.items[order_nr][0].status == 0 %}
+                                <button class="button is-warning" onclick="manage_order('1,{{order_nr}}')">Shipped</button>
+                                <button class="button is-danger" onclick="manage_order('3,{{order_nr}}')">Refund</button>
+                                {% elif ns.items[order_nr][0].status == 1 %}
+                                <!-- Can be used if you want the buttons to just be disabled and not disappear
```

Comment:
Writing comments like this passes them on to the user of the website. I think we should use Jinjas dissable functionality to write comments like this
```html
<!-- This comment will be passed to the user -->
{#
This comment will not be passed to the end user since Jinja will remove it
#}
```